lib/active_model_serializers/logging.rb

```diff
@@ -0,0 +1,112 @@
+##
+# ActiveModelSerializers::Logging
+#
+#   https://github.com/rails/rails/blob/280654ef88/activejob/lib/active_job/logging.rb
+#
+module ActiveModelSerializers::Logging
```

Comment:
Any special reason to use `ActiveModelSerializers::Logging` over a nested module? I think the more verbose is prefered http://blog.elpassion.com/ruby-gotchas/. Makes sense?